repositories-preset-wikimedia.txt

```diff
@@ -85,6 +85,7 @@ mediawiki/extensions/SecureLinkFixer
 mediawiki/extensions/SecurePoll
 mediawiki/extensions/SiteMatrix
 mediawiki/extensions/SpamBlacklist
+mediawiki/extensions/StopForumSpam
```

Comment:
I don't know much about this extension, is it installed on any WMF wikis?

Reply:
It's allegedly used by a handful of wikis right now: https://wikiapiary.com/wiki/Extension:StopForumSpam

It's also on Wikimedia beta right now, in report-only mode, with the eventual goal of deploying to production (or at least to a few projects), hopefully sooner than later: https://phabricator.wikimedia.org/T273220